src/main/java/it/pagopa/transactions/client/PaymentGatewayClient.java

```diff
@@ -733,6 +734,14 @@ public Mono<RedirectUrlResponseDto> requestRedirectUrlAuthorization(
 
     }
 
+    private String shortenRedirectPaName(@Nullable String paName) {
+        if (paName == null || paName.length() <= 70) {
+            return paName;
+        } else {
+            return paName.substring(0, 69) + "…";
```

Comment:
```suggestion
            return paName.substring(0, 67) + "...";
```

Reply:
I know but I would avoid use special characters that will go to external entities... maybe we can truncate to 70 characters without adding any trailing character

Reply:
Fixed in e23847b7